### ISSUE TITLE
StoryPromo title should not have a preceding comma

### ISSUE THREAD

**Issue by mykeels** [bbc/simorgh#6064]
**Describe the bug**
In the CPS Related Content component, [this comment](https://github.com/bbc/simorgh/pull/6040/files/c0dfdb69b34bd2ea809b535b13b2836f1c100a4a#r401012411) points out that the Headline text is prefixed with a comma.

The format should be **"[type], [text]"**, e.g. **"Video, Meet boys who dey convert cassava to electricity"** but because the type is unavailable, it becomes **", Meet boys who dey convert cassava to electricity"**

This happens because the `promos[0].media` object is not present in the test data, and adding it fixes the issue.

**To Reproduce**
Steps to reproduce the behaviour:

1. Go to https://bbc.github.io/simorgh/?path=/story/containers-cps-related-content-canonical--pidgin-ltr
2. Right-click on **Meet boys who dey convert cassava to electricity**
3. Inspect Element
4. Expand the <a> tag to reveal the `<span>` tags within it, as per the screenshot below
5. Notice the `span.VisuallyHiddenText-sc-1yjwwa5-0` element has a comma, and is hidden. This will affect screen readers.

![image](https://user-images.githubusercontent.com/11996508/78134345-e3044880-7417-11ea-90c5-83688fbbefd6.png)

**Expected behaviour**
If there is no media object, or if the media type cannot be inferred somehow, there should be no preceding text.

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Testing notes**

- [ ] This bug fix is expected to need manual testing.

**Additional context**
